plugins/ngrok/auth_credentials.go

```diff
@@ -61,8 +48,7 @@ func Credentials() schema.CredentialType {
 
 func ngrokConfig(in sdk.ProvisionInput) ([]byte, error) {
 	config := Config{
-		AuthToken: in.ItemFields[fieldname.Authtoken],
-		APIKey:    in.ItemFields[fieldname.APIKey],
+		AuthToken: in.ItemFields[fieldname.AuthToken],
```

Comment:
This should be `fieldname.Authtoken` because ngrok calls this secret `authtoken` as opposed to `Auth Token`